lib/Sips/Office/OfficeRequest.php

```diff
@@ -36,92 +34,91 @@ abstract protected function getMethod();
      *  }
      * }
      */
-    abstract protected function getFields();
+    abstract protected function getFields() : array;
 
-    /** @return string */
-    public function getSipsUri()
+    public function getSipsUri() : string
     {
         return $this->sipsUri;
     }
 
-    public function setBaseUrl($baseUrl)
+    public function setBaseUrl(string $baseUrl) : void
     {
         $uri = $baseUrl . $this->getMethod();
         $this->validateUri($uri);
         $this->sipsUri = $uri;
     }
 
     /** @return string */
-    public function getShaSign()
+    public function getShaSign() : string
     {
         return $this->shaComposer->compose($this->toArray());
     }
 
-    public function setParameter($name, $value)
+    /**
+     * @param Mixed $value
```

Comment:
Parameter type 'mixed' should be lowercase.

```suggestion
     * @param mixed $value
```